compare-master-to-stable.js

```diff
@@ -9,72 +9,72 @@ var Q = require('q');
 var _ = require('lodash');
 var semver = require('semver');
 
-var exec = function (cmd) {
-  return function () {
+var exec = function(cmd) {
+  return function() {
     var args = Array.prototype.slice.call(arguments, 0);
     args.unshift(cmd);
     var fullCmd = util.format.apply(util, args);
-    return Q.nfcall(cp.exec, fullCmd).then(function (out) {
+    return Q.nfcall(cp.exec, fullCmd).then(function(out) {
       return out[0].split('\n');
     });
   };
 };
 
-var andThen = function (fn, after) {
-  return function () {
+var andThen = function(fn, after) {
+  return /* @this */ function() {
     return fn.apply(this, arguments).then(after);
   };
 };
 
-var oneArg = function (fn) {
-  return function (arg) {
+var oneArg = function(fn) {
+  return function(arg) {
     return fn(arg);
   };
 };
 
-var oneLine = function (lines) {
+var oneLine = function(lines) {
   return lines[0].trim();
 };
 
-var noArgs = function (fn) {
-  return function () {
+var noArgs = function(fn) {
+  return function() {
     return fn();
   };
 };
 
-var identity = function (i) { return i; };
+var identity = function(i) { return i; };
 
 // like Q.all, but runs the commands in series
 // useful for ensuring env state (like which branch is checked out)
-var allInSeries = function (fn) {
-  return function (args) {
+var allInSeries = function(fn) {
+  return function(args) {
     var results = [];
     var def;
     while (args.length > 0) {
-      (function (arg) {
+      (function(arg) {
         if (def) {
-          def = def.then(function () {
+          def = def.then(function() {
             return fn(arg);
           });
         } else {
           def = fn(arg);
         }
-        def = def.then(function (res) {
+        def = def.then(function(res) {
           results.push(res);
         });
-      }(args.pop()));
+      })(args.pop());
```

Comment:
You don't *have to* (there is the "any" option) but it's better to be
consistent.

On Mon, Aug 1, 2016 at 4:30 PM, Georgios Kalpakas <notifications@github.com>
wrote:

> In compare-master-to-stable.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72986746>:
>
> >            results.push(res);
> >          });
> > -      }(args.pop()));
> > +      })(args.pop());
>
> Aha, I didn't know you need to choose one or the other with ESLInt.
> (I prefer outside, but I think it is a personal preference. Inside is
> fine.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/a35c7b1049edc8033c0e3a6eaeeda6cd34c99e73#r72986746>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUngksIxUA4vy4b7kjKplGlXUwv6Fvks5qbgMDgaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski